build.gradle.kts

```diff
@@ -110,10 +114,59 @@ publishing {
     repositories {
         maven {
             url = uri(spaceMavenRepositoryUrl)
+            name = "space"
             credentials {
                 username = System.getenv("SPACE_USERNAME") ?: spaceUsername
                 password = System.getenv("SPACE_PASSWORD") ?: spacePassword
             }
         }
     }
 }
+
+signing {
+    useGpgCmd()
```

Comment:
Sonatype accepts only signed packages. Currently local signing is configured. 

Requires extra pgp Gradle properties definition on the local machine:

```
signing.gnupg.executable=
signing.gnupg.useLegacyGpg=
signing.gnupg.homeDir=
signing.gnupg.keyName=
signing.gnupg.passphrase=
```